functions.php

```diff
@@ -186,7 +186,7 @@ function wp_autoupdates_is_plugins_auto_update_enabled() {
  * @return bool True if themes auto-update is enabled, false otherwise.
  */
 function wp_autoupdates_is_themes_auto_update_enabled() {
-	$enabled = ! defined( 'WP_DISABLE_THEMES_AUTO_UPDATE' ) || ! WP_DISABLE_THEMES_AUTO_UPDATE;
+	$enabled = true;
```

Comment:
Instead of setting `$enabled` to true, would be cleaner to just pass `true` to the filter.

Reply:
And keep mentioning $enabled in the inline docs?

Reply:
yes